src/zero/environments.clj

```diff
@@ -68,6 +68,10 @@
                             (->> (assoc m kw)))))]
     (reduce make {} ["development" "production"])))
 
+(def registered-envs
+  "Register a list of the available ENVIRONMENT names here."
+  ["development", "production"])
+
```

Comment:
I don't understand registered-envs.

Reply:
It's used in `once.clj` module, for creating a map between environments and their service account paths. But since now we have made the paths to service accounts identical to each other in the environments.clj, not sure if we really need this anymore...

Reply:
I'm also wondering this 🤔

Reply:
Per the video chat, that's purely me attempting to standardizing the envs and distinguishing depoy-able envs from project envs. We probably don't need this anymore. I will create a PR to pipeline-config to update this.